### PR TITLE
8th Transaltion 1114 Words

### PR DESCRIPTION
**Pull request by cherryloua**
Hi everyone,

Please, take some time again to review my PR. Thank you!

### CODE REVIEW

File: book/08-customizing-git/sections/config.asc [hunk 1]
```diff
@@ -20,12 +20,11 @@ Kung dadaanan mo ang `--system` na opsyon sa `git config`, binabasa at sinulat n
 Ang susunod na lugar na titingnan ng Git ay ang `~/.gitconfig` (o `~/.config/git/config`) na file, kung saan ay tiyak sa bawat gumagamit.
 Maaaring gawin ng Git na bumasa at sumulat sa file na ito sa pamamagitan ng pagpapasa ng `--global` na opsyon.
 
-Finally, Git looks for configuration values in the configuration file in the Git directory (`.git/config`) of whatever repository you're currently using.
-Sa katapusan, hinahanap ng Git ang mga value ng konpigurasyon sa konpigurasyon na file sa direktoryo ng Git (`.git/config`) ng anumang repositoryo na kasalukuyang ginagamit mo.
-Ang mga halagang ito ay tiyak sa isang solong repositoryo, at kumakatawan sa pagpasa ng `--local` na opsyon hanggang `git config`.
-(Kung hindi mo tukuyin kung aling antas ang gusto mong magtrabaho, ito ang default.)
+Panghuli, hinahanap ng Git ang mga halaga ng kompigurasyon sa file ng kompigurasyon sa direktoryo ng Git (`.git/config`) na anomang repositoryo na kasalukuyang ginagamit mo.
```
Comment (mind-blast) [inline book/08-customizing-git/sections/config.asc]: anomang to anumang

File: book/08-customizing-git/sections/config.asc [hunk 2]
```diff
@@ -26,26 +26,26 @@ Ang mga halagang ito ay tiyak sa isang solong repositoryo, at kumakatawan sa pag
...
-==== Basic Client Configuration
+==== Basic Client Kompigurasyon
 
-The configuration options recognized by Git fall into two categories: client-side and server-side.
-The majority of the options are client-side -- configuring your personal working preferences.
-Many, _many_ configuration options are supported, but a large fraction of them are useful only in certain edge cases; we'll cover just the most common and useful options here.
-If you want to see a list of all the options your version of Git recognizes, you can run
+Ang mga pagpipilian sa kompigurasyon na kinikilala ng Git ay may dalawang kategorya: client-side at server-side.
+Ang karamihan sa mga pagpipilian ay client-side -- i-kompigura ang mga personal na kagustuhan sa iyong ginagawa.
+Karamihan, ang _many_ mga opsyon sa kompigurasyon ay sinusuportahan, ngunit ang isang malaking bahagi nito ay magagamit sa piling mga pagkakataon lamang; sinasaklaw lamang namin ang pinaka-karaniwan at kapaki-pakinabang na mga pagpipilian dito.
```
Comment (mind-blast) [inline book/08-customizing-git/sections/config.asc]: many to maraming

File: book/08-customizing-git/sections/config.asc [hunk 3]
```diff
@@ -50,23 +50,23 @@ Maaari mo ring mahanap ang materyal ng reperensya sa http://git-scm.com/docs/git
 ===== `core.editor`
 
 ((($EDITOR)))((($VISUAL, see $EDITOR)))
-By default, Git uses whatever you've set as your default text editor via one of the shell environment variables `VISUAL` or `EDITOR`, or else falls back to the `vi` editor to create and edit your commit and tag messages.
-To change that default to something else, you can use the `core.editor` setting:
+Bilang default, ginagamit ng Git ang kahit anong itinakda mo bilang iyong default na editor ng teksto sa pamamagitan ng isa sa mga shell enviroment variable `VISUAL` o` EDITOR`, o kaya ay bumabalik sa editor ng `vi` upang lumikha at mag-edit ng iyong na i-commit at na tag ng mga mensahe.
```
Comment (mind-blast) [inline book/08-customizing-git/sections/config.asc]: tag ng mga mensahe to tag na mga mensahe

File: book/08-customizing-git/sections/config.asc [hunk 4]
```diff
@@ -50,23 +50,23 @@ Maaari mo ring mahanap ang materyal ng reperensya sa http://git-scm.com/docs/git
...
-Now, no matter what is set as your default shell editor, Git will fire up Emacs to edit messages.
+Sa ngayon, hindi mahalaga kung ano ang itinakda bilang iyong default na editor ng shell, ang Git ay gagamitin ang Emacs upang mag-edit ng mga mensahe.
 
 ===== `commit.template`
 
 (((commit templates)))
-If you set this to the path of a file on your system, Git will use that file as the default initial message when you commit.
-The value in creating a custom commit template is that you can use it to remind yourself (or others) of the proper format and style when creating a commit message.
+Kung itinakda mo ito sa path ng file sa iyong sistema, ang Git ay gagamitin ang file bilang unang mensahe na default kapag ikaw ay nag-commit.
+Ang halaga sa paglikha ng custom commit template ay maaari itong gamitin upang paalalahanan ang iyong sarili (o ang iba) ng tamang pormat at istilo kapag lumilikha ng mensahe na i-commit.
```
Comment (mind-blast) [inline book/08-customizing-git/sections/config.asc]: custom to pasadyang

File: book/08-customizing-git/sections/config.asc [hunk 5]
```diff
@@ -70,32 +70,33 @@ Halimbawa, tignan ang template file sa `~/.Gitmessage.txt` na mukhang ganito:
...
+Upang sabihin sa Git na gamitin ito bilang default na mensahe na lumilitaw sa iyong editor kapag nagpatakbo ka ng `git commit`, i-set ang `commit.template` na halaga ng kompigurasyon:
+
 [source,console]
 ----
 $ git config --global commit.template ~/.gitmessage.txt
 $ git commit
 ----
 
-Then, your editor will open to something like this for your placeholder commit message when you commit:
+Pagkatapos, magbubukas ang iyong editor sa isang bagay na tulad nito para sa iyong placeholder ng mensahe ng commit kapag nag-commit ka:
```
Comment (mind-blast) [inline book/08-customizing-git/sections/config.asc]: placeholder to paglalagyan

**Comment by mind-blast** [comment]
Please find time to check suggested words. Thank you,

**Comment by cherryloua** [comment]
Thanks for the review @robin-ho 

Hi @severinolorillajr I'm done with the changes. Thanks!

**Pull request merged**